fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/query/plan/temp/view/SortComparison.java

```diff
@@ -74,7 +74,8 @@ public String typelessString() {
     }
 
     @Override
-    public int planHash() {
+    public int planHash(PlanHashKind hashKind) {
+        // Right?
```

Comment:
it's ok -- this class is being phased out.

Reply:
Done.